Sources/EmbraceCrash/EmbraceCrashReporter.swift

```diff
@@ -55,16 +55,20 @@ public final class EmbraceCrashReporter: NSObject, CrashReporter {
             return
         }
 
-        if ksCrash.userInfo == nil {
-            ksCrash.userInfo = [:]
+        var crashInfo: [AnyHashable: Any] = [:]
+
+        if ksCrash.userInfo != nil {
+            crashInfo = ksCrash.userInfo
```

Comment:
What is this line trying to achieve? Is this to defend against some other addition to the ksCrash.userInfo outside of this class?

If necessary, this would be shorthand:
```
var crashInfo = ksCrash.userInfo ?? [:]
```